unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BackpackEditorHUDV2/AvatarSlot/AvatarSlotComponentView.cs

```diff
@@ -27,7 +27,7 @@ public class AvatarSlotComponentView : BaseComponentView<AvatarSlotComponentMode
         [SerializeField] internal Image focusedImage;
         [SerializeField] internal Image selectedImage;
         [SerializeField] private GameObject emptySlot;
-        [SerializeField] private GameObject hiddenSlot;
+        [SerializeField] internal GameObject hiddenSlot;
         [SerializeField] internal RectTransform tooltipContainer;
         [SerializeField] internal TMP_Text tooltipCategoryText;
         [SerializeField] internal TMP_Text tooltipHiddenText;
```

Comment:
I'm not sure if we should continue setting our private variables as `internal` only because we want to check them from our tests. If we make our tests check the functionality instead of each piece of code, we should be able to have all the internal variables as private.

Lets discuss this also with @lorux0 and lets reach a decision, because I think it is something that is happening a lot overall and it would be fine to have an agreement as team 👍

Reply:
I think that in this case it is correct. Because the GameObject/Components's state is the result/output of the processing of the view in each test case. Otherwise, there is nothing else to validate.
It is slightly different when we try to expose internal methods or internal fields just for the tests, because they are not part of the validation result.